### PR TITLE
Documentation updates: vcenter directory

### PR DESCRIPTION
**Pull request by gsperry2011**
Initial documentation of Puppet resources.

### CODE REVIEW

File: manifests/vcenter/dvs_portgroups.pp [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+#
+# @param dvswitch
+#   Name of the dvswitch we will add portgroups on.
+# @param dvs_pgs
+#   Hash of portgroup and VLAN IDs.
+#
+# @example Basic usage
+#   powercli::vcenter::dvs_portgroups {'my-vcenter.fqdn.tld':
+#     dvswitch => 'my_dvswitch_name_here',
+#     $dvs_pgs => { 'vlan_10_portgroup_name_here' => 10, 'vlan_11_portgroup_name_here' => 11 }
```
Comment (nmaludy) [inline manifests/vcenter/dvs_portgroups.pp]: these should be on their own line(s)

Comment (gsperry2011) [reply manifests/vcenter/dvs_portgroups.pp]: fixed

File: manifests/vcenter/dvs_portgroups.pp [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+#   Name of the dvswitch we will add portgroups on.
+# @param dvs_pgs
+#   Hash of portgroup and VLAN IDs.
+#
+# @example Basic usage
+#   powercli::vcenter::dvs_portgroups {'my-vcenter.fqdn.tld':
+#     dvswitch => 'my_dvswitch_name_here',
+#     $dvs_pgs => {
+#       'vlan_10_portgroup_name_here' => 10,
+#       'vlan_11_portgroup_name_here' => 11 }
```
Comment (nmaludy) [inline manifests/vcenter/dvs_portgroups.pp]: trailing } should be on its own line with a comm after it.

basically this whole thing should look like:

```
   powercli::vcenter::dvs_portgroups {'my-vcenter.fqdn.tld':
     dvswitch => 'my_dvswitch_name_here',
     dvs_pgs => {
       'vlan_10_portgroup_name_here' => 10,
       'vlan_11_portgroup_name_here' => 11,
      },
   }
```

Comment (gsperry2011) [reply manifests/vcenter/dvs_portgroups.pp]: fixed

**Review by nmaludy — APPROVED** [review]

**Pull request merged**